localization/english/traits/elf_race_traits_l_english.yml

```diff
@@ -1,50 +1,50 @@
 
  trait_elf_tier_1:0 "REPLACE ME!!!!!!!!"
 
  # Elf
  trait_elf:0 "Elf"
- trait_elf_desc:0 "\n | Race Of The Low Realms | \n \n Elves are the common people of all realms connected to the [divine_spark|E]. Its life energy fills them granting strong graceful forms, longer lifespans, and to some even the ability to wield [spark|E]. \n \n An Elf's higher alignment with the Spark makes them more susceptible to its will."
+ trait_elf_desc:0 "\n | Race Of The Low Realms | \n \n Elves are the common people of all realms connected to the [divine_spark|E]. Its life energy fills them granting strong graceful forms, longer lifespans, and to some, the ability to wield the [spark|E]. \n \nAn Elf's higher alignment with the Spark makes them more susceptible to its will."
 
  trait_elf_1:0 "Elf"
  trait_elf_1_desc:0 "
- trait_elf_1_character_desc:0 "[ROOT.Char.GetFirstNameNoTooltip] is yet a novice in cultivating spark"
- trait_elf_2_character_desc:0 "[ROOT.Char.GetFirstNameNoTooltip] is quite adept in the spark, and is well respected by their peers"
+ trait_elf_1_character_desc:0 "[ROOT.Char.GetFirstNameNoTooltip] is yet a novice in cultivating spark."
+ trait_elf_2_character_desc:0 "[ROOT.Char.GetFirstNameNoTooltip] is quite adept in the spark, and is well respected by their peers."
  trait_elf_3_character_desc:0 "[ROOT.Char.GetFirstNameNoTooltip] has walked many roads and seen much of the world."
  trait_elf_2:0 "Empowered Elf"
  trait_elf_3:0 "Pinnacle Elf"
  
  trait_track_elf_spark_alignment:0 "Spark Alignment"
- trait_track_elf_spark_alignment_desc:0 "Spark alignment is simply one's strength in manifestation of the Divine Spark's life energy. To Ascend to a higher state of being, elves must solidify Spark Energy into themselves, which in turn transforms them physically and spiritually to be more efficient vessels of Spark"
+ trait_track_elf_spark_alignment_desc:0 "Spark alignment is simply one's strength in manifestation of the Divine Spark's life energy. To Ascend to a higher state of being, elves must solidify Spark Energy into themselves, which in turn transforms them physically and spiritually to be more efficient vessels of Spark."
 
  # High Elf
  trait_high_elf:0 "High Elf"
- trait_high_elf_desc:0 "| Race Of The Low Realms | \n \n High Elves represent the peak race of the Lower Realms. Proud Elves whom fortify their bloodlines for generations, attain strong alignment with the Spark, or a mix of both. \n \n Their magics powerful, their forms strong, and their beauty radiant, they stand at the threshold of ascension."
+ trait_high_elf_desc:0 "| Race Of The Low Realms | \n \nHigh Elves represent the peak race of the Lower Realms. Proud elves who have fortified their bloodline for generations nad have an excellent grasp over the [spark|E]. \n \nTheir magics powerful, their forms strong, and their beauty radiant, they stand at the threshold of ascension."
```

Comment:
elves should be capitalized I believe as its a race name. 

also "nad" => "and"